pyiron/atomistics/structure/atoms.py

```diff
@@ -1553,6 +1590,14 @@ def plot3d(
             vector_color (numpy.ndarray): Colors for the vectors (only available with vector_field). (Default is None,
                 vectors are colored by their direction.)
             magnetic_moments (bool): Plot magnetic moments as 'scalar_field' or 'vector_field'.
+            view_plane (numpy.ndarray): A Nx3-array (N = 1,2,3); the first 3d-component of the array specifies
+                which plane of the system to view (for example, [1, 0, 0], [1, 1, 0] or the [1, 1, 1] planes), the
+                second 3d-component (if specified, otherwise [1, 0, 0]) gives the horizontal direction, and the third
+                component (if specified) is the vertical component, which is ignored and calculated internally. The
+                orthonormality of the orientation is internally ensured, and therefore is not required in the function
+                call. (Default is np.array([0, 0, 1]), which is view normal to the x-y plane.)
+            distance_from_camera (float): Distance of the camera from the structure. Higher = farther away.
```

Comment:
I see that setting `distance_from_camera=0` raises no error and NGLView shows nothing. So we could add

```
if distance_from_camera <= 0:
    raise ValueError('distance_from_camera must be a positive integer')
```

Or we could simply do:
```
if distance_from_camera <= 0:
    warnings.warn('distance_from_camera changed to 1')
    distance_from_camera = 1
```

I know this sounds stupid, but the most complicated problems often come from stupid mistakes...